text/0155-external-modules.md

```diff
@@ -0,0 +1,106 @@
+all available Otel components together with some visibility into their quality.
+
+## Explanation
+
+[OpenTelemetry Registry](https://opentelemetry.io/registry/) serves as a central catalogue of all known Otel components,
+both provided by core maintainers of the project and any third party.
+
+In order for a component to be included into Registry its authors have to fill a [self-assessment form](#registry-self-assessment-form).
+
+Registry should allow a clear visibility if component came from a repository in OpenTelemetry organisation.
+
+A component can be removed from the Registry if any declaration from the self-assessment form is violated and not remedied
+in a timely manner, provisionally one or two weeks.
+
+## Internal details
+
+We distinguish the following sources of OpenTelemetry components and integrations.
+
+### Native or built-in instrumentations
+
+Any library or framework can use OpenTelemetry API to natively produce Telemetry.
+We encourage library authors to submit their library for inclusion into OpenTelemetry Registry.
+
+### Core components
+
+Otel SIGs may provide instrumentations for any libraries that are deemed important enough by SIG’s maintainers.
+By doing this the SIG maintainers commit to support these instrumentations (including future versions of the library),
+provide updates/fixes, including security patches and guarantee that they comply with Otel semantic conventions and best practices.
+
+Depending on the SIG these core instrumentations may share repository, infrastructure and maintainers with
+OpenTelemetry API/SDK implementation for this language or be separate.
+
+All core instrumentations must be included into Otel Registry following the usual process described above.
+
+### Contrib components
+
+Any language SIG may have one or more “contrib” repos containing components contributed by external developers.
+This repository may have a separate set of approvers/maintainers than the core API/SDK repo.
+SIG maintainers are encouraged to have a simplified process to promote contributors to approver/maintainer role in this repository.
+Contrib repository may leverage the CODEOWNERS functionality of GitHub to assign maintainers to individual packages
+even if this means granting write permissions to the whole repo.
+The goal should be to distribute the load of reviewing PRs and accepting changes as much as possible.
+
+All components in contrib repository are expected to be included into Otel Registry following the usual process described above.
+
+It should be very easy for external contributors to include their components into contrib repo as opposed to hosting them separately.
+This way they can reuse existing infrastructure for testing, publishing, security scanning etc.
+Also this will greatly simplify responsibility transfer between different maintainers if their priorities change. It also promotes the development
```

Comment:
```suggestion
This will also greatly simplify responsibility transfer between different maintainers if their priorities change. It also promotes the development
```